src/main/app.ts

```diff
@@ -61,6 +61,8 @@ app.use(cookieEncrypter(config.get('secrets.cmc.encryptionKey'), {
   }
 }))
 
+// Web Chat
+app.use('/webchat', express.static(path.join(__dirname, '/node_modules/@hmcts/ctsc-web-chat/assets')))
```

Comment:
If you have copied everything into our folders in the above gulp file you shouldn't need to point to the node_modules folder. It should all be part of our files?

Reply:
Shouldn't we use from the library itself rather than copying locally? We don't want to end up different versions in our local.

Reply:
every time you do a yarn install then do a gulp it should copy and over-ride the files to the correct folders. We may need to add in a script to clear out the old ones. You normally only get problems if they delete files from their repo and we don't clear it up with gulp

Reply:
👍